<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="80" started-at="2015-04-12T15:02:03Z" finished-at="2015-04-12T15:02:03Z">
    <groups>
      <group name="1">
        <method signature="testStatic.test0002()[pri:0, instance:test.com.mm.android.lc.testStatic@1d43df5]" name="test0002" class="test.com.mm.android.lc.testStatic"/>
        <method signature="testStatic.test0003()[pri:0, instance:test.com.mm.android.lc.testStatic@1d43df5]" name="test0003" class="test.com.mm.android.lc.testStatic"/>
      </group> <!-- 1 -->
    </groups>
    <test name="testStatic" duration-ms="80" started-at="2015-04-12T15:02:03Z" finished-at="2015-04-12T15:02:03Z">
      <class name="test.com.mm.android.lc.testStatic">
        <test-method status="FAIL" signature="test0002()[pri:0, instance:test.com.mm.android.lc.testStatic@1d43df5]" name="test0002" duration-ms="27" started-at="2015-04-12T23:02:03Z" finished-at="2015-04-12T23:02:03Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at test.com.mm.android.lc.testStatic.test0002(testStatic.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test0002 -->
        <test-method status="SKIP" signature="test0003()[pri:0, instance:test.com.mm.android.lc.testStatic@1d43df5]" name="test0003" duration-ms="6" started-at="2015-04-12T23:02:03Z" depends-on-methods="test.com.mm.android.lc.testStatic.test0002" finished-at="2015-04-12T23:02:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test0003 -->
      </class> <!-- test.com.mm.android.lc.testStatic -->
    </test> <!-- testStatic -->
  </suite> <!-- Suite -->
</testng-results>
